src/constants.js

```diff
@@ -1,4 +1,4 @@
-export const MOUNT_SELECTOR = 'table.files'
+export const MOUNT_SELECTOR = '.Details [role="grid"]'
```

Comment:
We may need to figure out a better selector or more robust selector for this. For now this seems to have worked for the (minimum) pages on GitHub that I navigated.